tasks/FindMyMates.tex

```diff
@@ -0,0 +1,52 @@
+\section{Find My Mates [Party Host]}
+
+The robot starts in a room and waits for someone to approach it. The guest knows the name of other guests at the party but doesn’t know how these people look like and where the people are. The robot will gather this information for the guest.
```

Comment:
I'd change it to this:

> The robots start in a predefined room waiting for an operator. The operator has only the list of guests but does not know them. The robot shall describe all the people in the list to the operator.

However, let's see if we can shrink it to a one-liner

Reply:
That sounds less mechanical, I like it.

Reply:
For the main description we can have something like,

__Find My Mates [Party Host]__
The robot fetches the information of the party guests for the operator who knows only the names of the guests.